### PR TITLE
pages*: update outdated pages

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kant — APPROVED** [review]
spanish translation: LGTM

File: pages/common/bat.md [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 - Syntax highlight a JSON file:
 
-`bat --language json {{file.json}}`
+`bat --language {{json}} {{file.json}}`
```
Comment (acuteenvy) [inline pages/common/bat.md:25]: If you change `json` to something else, the file won't be treated as JSON.
```suggestion
`bat --language json {{path/to/file.json}}`
```

File: pages/common/curl.md [hunk 2]
```diff
@@ -30,7 +30,7 @@
 
 - Pass a username and password for server authentication:
 
-`curl --user myusername:mypassword {{http://example.com}}`
+`curl --user {{myusername:mypassword}} {{http://example.com}}`
```
Comment (acuteenvy) [inline pages/common/curl.md:33]:
```suggestion
`curl --user {{username}} {{http://example.com}}`
```

File: pages/common/curl.md [hunk 3]
```diff
@@ -30,7 +30,7 @@
 
 - Pass a username and password for server authentication:
```
Comment (acuteenvy) [inline pages/common/curl.md:31]: `man curl`
> ***-u, --user <user:password>***
> Specify the user name and password to use for server authentication.
> If you simply specify the user name, curl prompts for a password.

That way the password won't get saved to shell history.
```suggestion
- Pass a username and prompt for a password to authenticate to the server:
```

File: pages.de/common/aws-ec2.md [hunk 4]
```diff
@@ -16,9 +16,9 @@
 
 `aws ec2 describe-volumes`
 
-- Liste Informationen zu einem bestimmten EC2 Volumen auf:
+- Löschen ein EC2 Volumen:
```
Comment (gutjuri) [inline pages.de/common/aws-ec2.md:19]:
```suggestion
- Lösche ein EC2 Volumen:
```

File: pages.de/common/aws-ec2.md [hunk 5]
```diff
@@ -16,9 +16,9 @@
...
 
-`aws ec2 describe-volume --volume-id {{volumen_id}}`
+`aws ec2 delete-volume --volume-id {{volumen_id}}`
 
 - Erstelle einen Snapshot, basierend auf einem EC2 Volumen:
```
Comment (gutjuri) [inline pages.de/common/aws-ec2.md:23]:
```suggestion
- Erstelle einen Snapshot basierend auf einem EC2 Volumen:
```

File: pages.de/common/cat.md [hunk 6]
```diff
@@ -9,16 +9,16 @@
...
 - Verkette mehrere Dateien und hänge das Ergebnis an eine Datei an:
 
-`cat {{pfad/zu/datei1}} {{pfad/zu/datei2}} >> {{pfad/zu/ziel_datei}}`
+`cat {{pfad/zu/datei1 pfad/zu/datei2 ...}} >> {{pfad/zu/ziel_datei}}`
 
-- Nummeriere alle ausgegebenen Zeilen:
+- Kopieren den Inhalt einer Datei ohne Pufferung in eine Ausgabedatei:
 
-`cat -n {{pfad/zu/datei}}`
+`cat -u {{/dev/tty12}} > {{/dev/tty13}}`
```
Comment (gutjuri) [inline pages.de/common/cat.md:20]: For me it says that -u is ignored: https://manned.org/cat.1

File: pages.de/common/cat.md [hunk 7]
```diff
@@ -9,16 +9,16 @@
...
-`cat {{pfad/zu/datei1}} {{pfad/zu/datei2}} > {{pfad/zu/ziel_datei}}`
+`cat {{pfad/zu/datei1 pfad/zu/datei2 ...}} > {{pfad/zu/ziel_datei}}`
 
 - Verkette mehrere Dateien und hänge das Ergebnis an eine Datei an:
 
-`cat {{pfad/zu/datei1}} {{pfad/zu/datei2}} >> {{pfad/zu/ziel_datei}}`
+`cat {{pfad/zu/datei1 pfad/zu/datei2 ...}} >> {{pfad/zu/ziel_datei}}`
 
-- Nummeriere alle ausgegebenen Zeilen:
+- Kopieren den Inhalt einer Datei ohne Pufferung in eine Ausgabedatei:
```
Comment (gutjuri) [inline pages.de/common/cat.md:18]:
```suggestion
- Kopiere den Inhalt einer Datei in eine Ausgabedatei ohne zu puffern:
```
Please do not use an online translator, I'm happy to help out with translations if pinged.

File: pages.de/common/cat.md [hunk 8]
```diff
@@ -9,16 +9,16 @@
...
 
-`cat -n {{pfad/zu/datei}}`
+`cat -u {{/dev/tty12}} > {{/dev/tty13}}`
 
-- Gib eine Datei inklusive aller unsichtbaren Leerzeichen aus (mit `M-` Präfix wenn sie nicht ASCII sind):
+- Schreib `stdin` in eine Datei:
```
Comment (gutjuri) [inline pages.de/common/cat.md:22]:
```suggestion
- Schreibe `stdin` in eine Datei:
```

File: pages.de/common/docker-compose.md [hunk 9]
```diff
@@ -9,15 +9,15 @@
...
 
-`docker compose up -d`
+`docker compose up --detach`
 
 - Starte alle Container. Erzeuge zugehörige Docker Images bei Bedarf neu:
 
 `docker compose up --build`
 
-- Starte alle Container unter Verwendung einer alternativen Compose Datei:
+- Starte alle Container durch Angabe eines Projektnamens unter Verwendung einer alternativen Compose Datei:
```
Comment (gutjuri) [inline pages.de/common/docker-compose.md:18]:
```suggestion
- Starte alle Container durch Angabe eines Projektnamens unter Verwendung einer alternativen Compose-Datei:
```

File: pages.de/common/docker-compose.md [hunk 10]
```diff
@@ -9,15 +9,15 @@
...
 
-`docker compose --file {{pfad/zu/verzeichnis}} up`
+`docker compose -p {{Projectnamens}} --file {{pfad/zu/verzeichnis}} up`
```
Comment (gutjuri) [inline pages.de/common/docker-compose.md:20]:
```suggestion
`docker compose -p {{Projektname}} --file {{pfad/zu/verzeichnis}} up`
```

File: pages.de/common/docker.md [hunk 11]
```diff
@@ -4,25 +4,25 @@
...
 - Stoppe oder starte einen existierenden Container:
 
 `docker {{start|stop}} {{container_name}}`
 
 - Lade ein Image herunter:
 
-`docker pull {{pfad/zu/image}}`
+`docker pull {{image}}`
+
+- Liste der bereits heruntergeladenen Imagen an:
```
Comment (gutjuri) [inline pages.de/common/docker.md:23]:
```suggestion
- Zeige eine Liste der bereits heruntergeladenen Images an:
```

File: pages.de/common/ffprobe.md [hunk 12]
```diff
@@ -5,7 +5,7 @@
 
 - Zeige alle verfügbaren Stream-Informationen einer Medien-Datei an:
 
-`ffprobe -v error -show_entries {{datei.mp4}}`
+`ffprobe -v error -show_streams {{datei.mp4}}`
 
 - Zeige Spieldauer an:
```
Comment (gutjuri) [inline pages.de/common/ffprobe.md:10]:
```suggestion
- Zeige die Spieldauer an:
```

File: pages.de/common/flutter.md [hunk 13]
```diff
@@ -3,9 +3,9 @@
 > Googles Cross-Platform Open-Source SDK.
 > Weitere Informationen: <https://github.com/flutter/flutter/wiki/The-flutter-tool>.
 
-- Zeige hilfe für einen bestimmten Befehl:
+- Initialisieren ein neues Flutter Projekt in einem gleichnamigen Verzeichnis:
```
Comment (gutjuri) [inline pages.de/common/flutter.md:6]:
```suggestion
- Initialisiere ein neues Flutter-Projekt in einem gleichnamigen Verzeichnis:
```

File: pages.de/common/flutter.md [hunk 14]
```diff
@@ -19,14 +19,14 @@
...
-
 - Starte Tests in einem Projekt vom Wurzelverzeichnes aus:
 
 `flutter test {{test/beispiel_test.dart}}`
 
 - Baue eine Release APK für die meisten modernen Smartphones:
 
 `flutter build apk --target-platform {{android-arm}},{{android-arm64}}`
+
+- Zeige hilfe für einen bestimmten Befehl:
```
Comment (gutjuri) [inline pages.de/common/flutter.md:30]:
```suggestion
- Zeige Hilfe für einen bestimmten Befehl:
```

File: pages.de/common/git-branch.md [hunk 15]
```diff
@@ -3,13 +3,13 @@
...
 > Weitere Informationen: <https://git-scm.com/docs/git-branch>.
 
-- Liste alle lokalen Branches auf. Der momentan aktive (ausgecheckte) Branch wird mit `*` markiert:
+- Liste alle Branches auf (Lokal und Remote). Der momentan aktive (ausgecheckte) Branch wird mit `*` markiert:
 
-`git branch`
+`git branch --all`
 
-- Liste alle Branches auf (Lokal und Remote):
+- Liste alle Branches welche ein spezifisches Git commit in ihre Geschichte enthalten:
```
Comment (gutjuri) [inline pages.de/common/git-branch.md:10]:
```suggestion
- Liste alle Branches auf, welche einen spezifischen Git-Commit in ihrer Historie enthalten:
```

File: pages.de/common/git-commit.md [hunk 16]
```diff
@@ -5,19 +5,19 @@
 
 - Committe gestagten Dateien zum Repository mit einer Nachricht:
```
Comment (gutjuri) [inline pages.de/common/git-commit.md:6]:
```suggestion
- Committe die gestagten Dateien mit einer Nachricht in das Repository:
```

File: pages.de/common/git-commit.md [hunk 17]
```diff
@@ -5,19 +5,19 @@
...
 
 `git commit --file {{pfad/zu/commit_nachricht_datei}}`
 
 - Stage alle modifizierten Dateien und committe sie mit einer Nachricht:
 
-`git commit -a -m "{{nachricht}}"`
+`git commit --all --message "{{nachricht}}"`
 
-- Committe alle gestagten Dateien und [S]igniere sie mit dem in `~/.gitconfig` definierten GPG Schlüssel:
+- Committe gestagten Dateien und signiere sie mit dem definierten GPG Schlüssel (oder die in der Konfigurationsdatei definierte, wenn kein Argument angegeben ist):
```
Comment (gutjuri) [inline pages.de/common/git-commit.md:18]:
```suggestion
- Committe gestagten Dateien und signiere sie mit dem definierten GPG Schlüssel (oder mit dem in der Konfigurationsdatei definierten, wenn kein Argument angegeben ist):
```

**Review by gutjuri — DISMISSED** [review]
Thanks for your PR. I have some suggestions.

**Comment by kbdharun** [comment]
> How do we want to cope with the rest of the outdated pages? I could create several new bulk PRs that update multiple pages, or I could check which page is outdated for multiple languages and open a PR per multi-language outdated page and open a PR per language with pages that isn’t outdated for multiple languages.

In the past, for major changes like this, we just focused on updating the English pages and left the translation pages to be updated by others. I would suggest doing the same as it adds up work when we modify translations too (instead I would recommend putting the effort into adding new pages or improving the English pages).

If you are still interested in fixing minor outdated translations then opening separate PRs for each language would indeed ease up the review process.

File: pages.de/common/curl.md [hunk 18]
```diff
@@ -28,9 +28,9 @@
 
 `curl --data {{'{"name":"karl-dieter"}'}} --header {{'Content-Type: application/json'}} {{http://beispiel.de/benutzer/1234}}`
 
-- Übergib einen Benutzernamen und Passwort für die Server-Authentifizierung:
+- Übergib einen Benutzernamen und frage  für ein Passwort für die Server-Authentifizierung:
```
Comment (acuteenvy) [inline pages.de/common/curl.md:31]:
```suggestion
- Übergib einen Benutzernamen und frage für ein Passwort für die Server-Authentifizierung:
```

File: pages.fr/common/bat.md [hunk 19]
```diff
@@ -18,11 +18,11 @@
 
 - Numérote toutes les lignes affichées :
 
-`bat -n {{fichier}}`
+`bat --number {{fichier}}`
 
 - Affiche le contenu d'un fichier JSON sur la sortie standard avec mise en valeur de la syntaxe :
 
-`bat --language json {{fichier.json}}`
+`bat --language {{json}} {{fichier.json}}`
```
Comment (acuteenvy) [inline pages.fr/common/bat.md:25]:
```suggestion
`bat --language json {{chemin/vers/fichier.json}}`
```

File: pages.id/common/bat.md [hunk 20]
```diff
@@ -18,11 +18,11 @@
 
 - Memberi nomor pada setiap baris keluaran:
 
-`bat -n {{berkas}}`
+`bat --number {{berkas}}`
 
 - Mencetak konten JSON dengan sintaks berwarna:
 
-`bat --language json berkas.json`
+`bat --language {{json}} {{berkas.json}}`
```
Comment (acuteenvy) [inline pages.id/common/bat.md:25]:
```suggestion
`bat --language json {{jalan/menuju/berkas.json}}`
```

File: pages.id/common/curl.md [hunk 21]
```diff
@@ -30,7 +30,7 @@
 
 - Memberikan nama pengguna dan kata sandi untuk otentikasi server:
 
-`curl --user myusername:mypassword {{http://contoh.com}}`
+`curl --user {{myusername}} {{http://contoh.com}}`
```
Comment (acuteenvy) [inline pages.id/common/curl.md:33]:
```suggestion
`curl --user {{username}} {{http://contoh.com}}`
```

File: pages.ja/common/curl.md [hunk 22]
```diff
@@ -30,7 +30,7 @@
 
 - サーバー認証のためユーザー名とパスワードを送る:
 
-`curl --user myusername:mypassword {{http://example.com}}`
+`curl --user {{myusername}} {{http://example.com}}`
```
Comment (acuteenvy) [inline pages.ja/common/curl.md:33]:
```suggestion
`curl --user {{username}} {{http://example.com}}`
```

File: pages.ko/common/curl.md [hunk 23]
```diff
@@ -30,7 +30,7 @@
 
 - 서버 인증을 위한 사용자 이름 및 비밀번호 전달:
 
-`curl --user myusername:mypassword {{http://example.com}}`
+`curl --user {{myusername}} {{http://example.com}}`
```
Comment (acuteenvy) [inline pages.ko/common/curl.md:33]:
```suggestion
`curl --user {{username}} {{http://example.com}}`
```

File: pages.zh/common/curl.md [hunk 24]
```diff
@@ -30,7 +30,7 @@
 
 - 使用用户名和密码，授权访问服务器：
 
-`curl --user myusername:mypassword {{http://example.com}}`
+`curl --user {{myusername}} {{http://example.com}}`
```
Comment (acuteenvy) [inline pages.zh/common/curl.md:33]:
```suggestion
`curl --user {{username}} {{http://example.com}}`
```

File: pages.zh_TW/common/curl.md [hunk 25]
```diff
@@ -30,7 +30,7 @@
 
 - 透過使用者名稱和密碼訪問伺服器：
 
-`curl --user myusername:mypassword {{http://example.com}}`
+`curl --user {{myusername}} {{http://example.com}}`
```
Comment (acuteenvy) [inline pages.zh_TW/common/curl.md:33]:
```suggestion
`curl --user {{username}} {{http://example.com}}`
```

**Review by acuteenvy — APPROVED** [review]

File: pages.de/common/curl.md (hunk 18 above)
Comment (gutjuri) [inline pages.de/common/curl.md:31]:
```suggestion
- Übergib einen Benutzernamen und frage nach einem Passwort für die Server-Authentifizierung:
```

File: pages.de/common/ab.md [hunk 26]
```diff
@@ -11,14 +11,14 @@
 
 `ab -n {{100}} -c {{10}} {{url}}`
 
+- Sende 100 HTTP POST Anfragen an eine URL, wobei eine JSON Belastung aus einer Datei verwendet wird:
```
Comment (gutjuri) [inline pages.de/common/ab.md:14]:
```suggestion
- Sende 100 HTTP POST Anfragen mit der in der angegebenen Datei gespeicherten Payload an eine URL:
```

File: pages.de/common/cat.md (hunk 6 above)
Comment (kbdharun) [reply pages.de/common/cat.md:20]: This flag is indeed ignored in some versions, but I wouldn't suggest removing it in `common` as it is still supported in some places i.e. https://manned.org/man/cat.1posix. Instead, we can remove it on the `linux` variant of the page alone. (Regarding `-n` I don't think we should add it in the `common` page, see the last section of above manpage for more information)

**Review by kbdharun — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Review by reinhart1010 — APPROVED** [review]
Not much of an issue for the Indonesian translation

**Review by blueskyson — APPROVED** [review]
Chinese pages look good. Though `username` can be translated later.

**Review by kbdharun — APPROVED** [review]

**Review by isaacvicente — APPROVED** [review]

**Pull request merged**